src/containers/forms/PlanForm/index.tsx

```diff
@@ -211,12 +211,12 @@ const PlanForm = (props: PlanFormProps) => {
                               : 'async-select-container'
                           }
                         >
-                          <Label for={`jurisdictions[${index}].id`}>Focus Area</Label>
+                          <Label for={`jurisdictions-${index}-id`}>Focus Area</Label>
                           <Field
                             required={true}
                             component={JurisdictionSelect}
                             name={`jurisdictions[${index}].id`}
-                            id={`jurisdictions[${index}].id`}
+                            id={`jurisdictions-${index}-id`}
```

Comment:
Changed all ids to be like this so that we avoid the square brackets in the id values - this was causing problems with enzyme.